internal/entities/user_management/bank/core.go

```diff
@@ -1 +1,34 @@
 package bank
+
+import (
+	"fmt"
+	"github.com/freshpay/internal/config"
+	"github.com/freshpay/internal/entities/user_management/utilities"
+)
+
+//CreateBank will create a new bank
+func CreateBank(bank *Detail,userId string)(err error){
+	bank.ID=utilities.CreateID(14)
+	bank.UserId=userId
+	fmt.Println("banK: ",bank)
+	if err=config.DB.Create(bank).Error; err!=nil{
+		return err
+	}
+	return nil
+}
+
+//GetBankById will return the bank by using the id
+func GetBankById(bank *Detail, id string)(err error){
+	if err = config.DB.Where("ID = ?", id).First(bank).Error; err != nil {
```

Comment:
create new file `bank/repo.go`. ideally core should not talk to DB directly. It should talk via `repo`

Reply:
Do similar changes everywhere